gemini_test.py

```diff
@@ -138,6 +139,33 @@ def get_email_data(self):
             self.log.warning("Failed to get gemini version as loader instance is not created")
             gemini_version = ""
 
+        try:
+            seed = self.params.get("gemini_seed")
+            if not seed:
+                seed_match = re.search(r"--seed (?P<seed>\d+) ", self.gemini_results["cmd"])
+                if seed_match:
+                    seed = seed_match.groupdict().get("seed")
+                else:
+                    seed = -1
+
+            # pylint: disable=no-member
+            self.test_config.argus_client().submit_gemini_results({
+                "gemini_command": self.gemini_results["cmd"],
+                "gemini_read_errors": self.gemini_results["results"].get("read_errors", -1),
+                "gemini_read_ops": self.gemini_results["results"].get("read_ops", -1),
+                "gemini_seed": seed,
+                "gemini_status": self.gemini_results["status"],
+                "gemini_version": gemini_version,
+                "gemini_write_errors": self.gemini_results["results"].get("write_errors", -1),
+                "gemini_write_ops": self.gemini_results["results"].get("write_ops", -1),
+                "oracle_node_ami_id": self.params.get("ami_id_db_oracle"),
+                "oracle_node_instance_type": self.params.get("instance_type_db_oracle"),
+                "oracle_node_scylla_version": self.cs_db_cluster.nodes[0].scylla_version if self.cs_db_cluster else "N/A",
+                "oracle_nodes_count": self.params.get("n_test_oracle_db_nodes"),
+            })
+        except Exception:  # pylint: disable=broad-except
+            self.log.warning("Error submitting gemini results to argus")
```

Comment:
please make sure we print the callstack here, otherwise it would be painful to debug.

BTW you last run ends here

Reply:
Yep, missed that one, added exception info.